composer.json

```diff
@@ -14,7 +14,7 @@
   "require": {
     "php": "^7.1",
     "illuminate/support": "5.0 - 5.8 | ^6.0",
-    "sentry/sdk": "^2.0"
+    "sentry/sdk": "^2.0.4"
```

Comment:
Is this needed in the context of this PR?

Reply:
Yes this is required to inlude the fixed/missing breadcrumb level `fatal`. 

See: https://github.com/getsentry/sentry-php/pull/902

Reply:
Ok actually I expected that the latest sentry sdk requires the latest sentry-php. But it just requires ^2.1.3 up from ^2.0.1 with this change. Which means that the required change might not be in there if a user explicitly wants something between 2.1.3 and 2.2.2.

Reply:
We could skip this version bumb until sentry/sdk bumps it's version of sentry-php. What do you suggest?

Reply:
I see. It's not optimal but what about we leave the requirement and do this until we can depend on 3.x?

```php
        // @TODO: Once we depend on `sentry-php` 3.0 we can remove this check and use `Breadcrumb::LEVEL_FATAL`.
        $fatal = defined(Breadcrumb::class . '::LEVEL_FATAL')
            ? Breadcrumb::LEVEL_FATAL
            : Breadcrumb::LEVEL_CRITICAL;

        switch (strtolower($level)) {
// ...
            case 'critical':
            case 'alert':
            case 'emergency':
                return $fatal;
// ...
        }
```

Reply:
Sound reasonable. I will add the suggested solution.